src/table.rs

```diff
@@ -402,18 +418,76 @@ impl Table {
             .reduce(|x, y| x + y)
             .unwrap()
     }
+
+    fn get_bet_increases_amount(players: &[Player]) -> Vec<i32> {
```

Comment:
I've looked at this function for a while, but I'm not getting what it's trying to do. Is the list it returns supposed to have negatives?

Reply:
Yeah its documented now, let me know what you think of those.

Reply:
It checks a lot more things now.

Reply:
Seems good. ;)